### ISSUE TITLE
Script can't not work well on openSUSE-Leap-15.2

### ISSUE THREAD

**Issue by 906341774** [Alxaline/Easy-GATE-Installation#3]
Hello, Mr Alxaline:
I am very grateful for your script trying to make Gate installation more easier, which help me out from the awful options of every components compiling. Moreover, there is few passages about installing Gate on openSUSE. Your work really give hope for a suse lover.

However, I am unlucky to occur some error when using the script. I have been run the script for two times. Because the installation took a long time, I run the script before sleep and wake up to see the progress next morning. 

I found that it almost occured error during making install Root, Geant4, Gate, like the following pictures.
![easy install Gate script error information _ 2022-04-14 094252](https://user-images.githubusercontent.com/65586854/163334029-f71bf483-4e60-41e3-9355-7559354382d6.png)

So I guess maybe some depencies in previous steps failed to install. Then, I set up for a try. Finally, I found that some of required packages in the scipt can't be found in openSUSE official repositories(Following Picture) nowaday. 
Here are some of them: 
xorg-x11-libX11-devel xorg-x11-libXpm-devel xorg-x11-proto-devel xorg-x11-libXext-devel.
![some required packages can't be found_2022-04-14 150745](https://user-images.githubusercontent.com/65586854/163332752-73aed456-449c-46fe-a246-50e83d121708.png)

Maybe it isn't the only reason of the failing. I will attempt to make my issue more clear. Yours work already make me benefit a lot. If you have rest time to give a solution, I'd like to try it until it work.

According to the previous issues, I left my enviornment information here, hope it is useful.
```shell
printenv
```
and the reasult is:
```shell
SHELL=/bin/zsh
PATH=/usr/local/bin:/bin:/usr/bin:/usr/local/bin:/bin:/usr/bin:/home/sony-c/bin:/usr/local/bin:/usr/bin:/bin:/usr/lib/mit/bin:/usr/lib/mit/sbin:/snap/bin:/usr/lib/mit/bin:/usr/lib/mit/sbin
HOME=/home/sony-c
USER=sony-c
LOGNAME=sony-c
TERM=xterm-256color
XDG_SESSION_ID=c2
XDG_RUNTIME_DIR=/mnt/wslg/runtime-dir
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
WSL_INTEROP=/run/WSL/29_interop
WSL_DISTRO_NAME=openSUSE-Leap-15.2
DISPLAY=172.19.0.1:0.0
SHLVL=1
PWD=/home/sony-c
OLDPWD=/mnt/g/Users/sony - c
CSHEDIT=emacs
MAIL=/var/spool/mail/sony-c
HOST=VAIO-SVE14AE13T
CPU=x86_64
HOSTNAME=VAIO-SVE14AE13T
HOSTTYPE=x86_64
OSTYPE=linux
MACHTYPE=x86_64-suse-linux
HISTSIZE=50000
LESSOPEN=lessopen.sh %s
LESSCLOSE=lessclose.sh %s %s
LESS=-M -I -R
LESSKEY=/etc/lesskey.bin
PAGER=less
LESS_ADVANCED_PREPROCESSOR=no
MORE=-sl
MINICOM=-c on
MANPATH=/usr/local/man:/usr/share/man
XKEYSYMDB=/usr/X11R6/lib/X11/XKeysymDB
XNLSPATH=/usr/X11R6/lib/X11/nls
COLORTERM=1
LANG=zh_CN.UTF-8
NAME=VAIO-SVE14AE13T
PULSE_SERVER=/mnt/wslg/PulseServer
WAYLAND_DISPLAY=wayland-0
WSLENV=WT_SESSION::WT_PROFILE_ID
WT_PROFILE_ID={914a0d3c-9db4-56f8-a8ac-65f4b7ea8b09}
WT_SESSION=4de98116-b563-4894-8249-ba73fe1dc2ed
http_proxy=http://192.168.1.2:7890
https_proxy=http://192.168.1.2:7890
ftp_proxy=http://192.168.1.2:7890
gopher_proxy=
socks_proxy=
SOCKS_PROXY=
SOCKS5_SERVER=
no_proxy=localhost,127.0.0.1
NO_PROXY=localhost,127.0.0.1
WINDOWMANAGER=xterm
XDG_DATA_DIRS=/usr/local/share:/usr/share:/var/lib/snapd/desktop
XDG_CONFIG_DIRS=/etc/xdg
G_BROKEN_FILENAMES=1
G_FILENAME_ENCODING=@locale,UTF-8,GB2312,GB18030,GBK,ISO-8859-1
MAN_KEEP_FORMATTING=yes
PROFILEREAD=true
LS_COLORS=no=00:fi=00:di=01;34:ln=00;36:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=41;33;01:ex=00;32:*.cmd=00;32:*.exe=01;32:*.com=01;32:*.bat=01;32:*.btm=01;32:*.dll=01;32:*.tar=00;31:*.tbz=00;31:*.tgz=00;31:*.rpm=00;31:*.deb=00;31:*.arj=00;31:*.taz=00;31:*.lzh=00;31:*.lzma=00;31:*.zip=00;31:*.zoo=00;31:*.z=00;31:*.Z=00;31:*.gz=00;31:*.bz2=00;31:*.tb2=00;31:*.tz2=00;31:*.tbz2=00;31:*.xz=00;31:*.avi=01;35:*.bmp=01;35:*.dl=01;35:*.fli=01;35:*.gif=01;35:*.gl=01;35:*.jpg=01;35:*.jpeg=01;35:*.mkv=01;35:*.mng=01;35:*.mov=01;35:*.mp4=01;35:*.mpg=01;35:*.pcx=01;35:*.pbm=01;35:*.pgm=01;35:*.png=01;35:*.ppm=01;35:*.svg=01;35:*.tga=01;35:*.tif=01;35:*.webm=01;35:*.webp=01;35:*.wmv=01;35:*.xbm=01;35:*.xcf=01;35:*.xpm=01;35:*.aiff=00;32:*.ape=00;32:*.au=00;32:*.flac=00;32:*.m4a=00;32:*.mid=00;32:*.mp3=00;32:*.mpc=00;32:*.ogg=00;32:*.voc=00;32:*.wav=00;32:*.wma=00;32:*.wv=00;32:
LS_OPTIONS=-N --color=tty -T 0
GPG_TTY=/dev/pts/0
ZSH=/etc/oh-my-zsh
LSCOLORS=Gxfxcxdxbxegedabagacad
ZSH_CACHE_DIR=/home/sony-c/.oh-my-zsh/cache
LC_ALL=en_US.UTF-8
_=/usr/bin/printenv
```

**Comment by Alxaline** [comment]
Hello,

I have set an openSUSE-Leap-15.2 distribution. 
Error appears during the root compilation. due to binutils. See the following issue :
https://github.com/afterstep/afterstep/issues/3

So a solution is to replace in the Makefile of the following folder :  GATE/root-6.20.04-build/AFTERIMAGE-prefix/src/AFTERIMAGE
`AR = ar clq` to `AR	= ar cq`

I added a line and this version will hopefully work for you : 
[Easy-GATE-Installation-master.zip](https://github.com/Alxaline/Easy-GATE-Installation/files/8498919/Easy-GATE-Installation-master.zip)


Best regards,
